package.json

```diff
@@ -5,9 +5,12 @@
   "main": "index.js",
   "scripts": {
     "test:open": "npm run lint && cypress open",
+    "pretest": "rm -rf cypress/reports/*",
     "lint": "eslint ./cypress/e2e/**/*.ts",
     "lint:fix": "npm run lint -- --fix",
-    "test": "cypress run"
+    "test": "cypress run",
+    "combine:reports": "mochawesome-merge cypress/reports/mocha/*.json > cypress/reports/report.json",
+    "generate:reports": "marge cypress/reports/report.json -f report -o cypress/reports"
```

Comment:
are these changes required in this step?

Reply:
this changes are for the  step 13

Reply:
Oh I see so you are including step 13 and 14 in this PR. that's ok but, then you need to add t he posttest script and attach an screenshot of both:

- HTML report
- console report